cmd/minishift/cmd/openshift/config.go

```diff
@@ -27,8 +27,8 @@ const (
 
 var configCmd = &cobra.Command{
 	Use:   "config SUBCOMMAND [flags]",
-	Short: "Displays or patches OpenShift configuration.",
-	Long:  "Displays or patches OpenShift master or node configuration. Patches are supplied in JSON format.",
+	Short: "Displays or updates the OpenShift configuration.",
```

Comment:
Sorry, I take this back. In other config command we use "modify" as well as "update". Let me take a look again and see what the best way to standardize.